recipes/hboard-widget/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "hboard-widget" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DataCanvasIO/HyperBoard/archive/refs/tags/{{ version }}.tar.gz
+  sha256: bd2d4f08937f0991c28b2489b0c11f69eda71dea6587a0debb89682581e1be03
+
+build:
+  noarch: python
+  number: 0
+  script: "cd hboard-widget && {{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6,<3.10
+    - jupyterlab >=3.0
+    - jupyter-packaging
+    - pip
+    - yarn
+  run:
+    - python >=3.6,<3.10
+    - hypernets >=0.2.5.1,<0.2.6
+    - ipywidgets >=7.6.0
+  run_constrained:
+    - woodwork <0.14.0
```

Comment:
I add `woodwork <0.14.0` to fix the error:
```
import: 'hboard_widget'
....
  File "/home/conda/staged-recipes/build_artifacts/hboard-widget_1647409605591/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.9/site-packages/featuretools/entityset/entityset.py", line 12, in <module>
    from featuretools.entityset import deserialize, serialize
  File "/home/conda/staged-recipes/build_artifacts/hboard-widget_1647409605591/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.9/site-packages/featuretools/entityset/serialize.py", line 7, in <module>
    from woodwork.serialize import typing_info_to_dict
ModuleNotFoundError: No module named 'woodwork.serialize'
```



I checked the version of `woodwork` in [CI log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=475537&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2728) ：
```
    woodwork:                      0.14.0-pyhd8ed1ab_0        conda-forge
```


I tried to reproduce this error on my machine by installing `woodwork==0.14.0` and then test by statement `from woodwork.serialize import typing_info_to_dict`, outputs is：
```
$ conda list |grep woodwork
woodwork                  0.14.0             pyhd8ed1ab_0    conda-forge
$ python -c "from woodwork.serialize import typing_info_to_dict"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'woodwork.serialize'
```
The error reappeared, then I try to install `woodwork==0.13.0` and the import statement works well.
I think this may be due to changes in the API of `woodwork==0.14.0`, but the `featuretools` have not been adapted yet.


Ref:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=475489&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2749